modules/openapi-generator/src/main/java/org/openapitools/codegen/ObjectWithTypeBooleans.java

```diff
@@ -0,0 +1,58 @@
+package org.openapitools.codegen;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ObjectWithTypeBooleans {
```

Comment:
can you please add a docstring explaining the purpose of this class? thanks.

Reply:
For your info, I think that this is a more sustainable way of generating things like code examples because:
- it allows any depth of indentation
- it works for all languages without difficult specific customization
- it doesn't require n number of nested inner imports